{"name": "Проверка авторизации с неверным паролем", "status": "passed", "description": "Создаем пользователя, авторизуемся с неверным паролем,             проверяем, что код ответа: 401, и текст ответа \"email or password are incorrect\"", "start": 1712483267119, "stop": 1712483267944, "uuid": "90aee225-be17-4402-bc29-228d6acdc3e6", "historyId": "b6f2108a3e9dc9ad27d6ff1dd16875a0", "testCaseId": "b6f2108a3e9dc9ad27d6ff1dd16875a0", "fullName": "tests.test_user.TestUser#test_login_user_unvalid_password_faild", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "DESKTOP-1VNC7DB"}, {"name": "thread", "value": "25008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}